examples/with-claims/src/test.context.jsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from "react";
```

Comment:
@porcellus missed this comment?

Reply:
We've touched on it on a call, but basically, this file is there to support the "log view" for the example. This is mainly intended to demo/visualize the events/API calls/etc. I made this to show how claims work under the hood. 
I think this could be useful in general, but not really as an example to follow, so I'll remove this later.